package.json

```diff
@@ -14,8 +14,8 @@
     "lint": "npm run lint:js && npm run lint:css",
     "lint:js": "eslint --ext .js,.vue .",
     "lint:css": "stylelint 'src/**/*.vue'",
-    "storybook": "start-storybook -p 6006",
-    "build-storybook": "build-storybook"
+    "storybook": "start-storybook -s ./public -p 6006",
```

Comment:
I don't think we want this change, we don't want to serve the static storybook in local dev